design-library/source/_patterns/01-atoms/10-footer-text/footer-image.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Could we rename this file to `footer-text.md`? If not, please rename the directory to "footer-image"